packages/terra-date-input/src/DateInput.jsx

```diff
@@ -606,7 +606,7 @@ class DateInput extends React.Component {
       <div className={DateInputMonthWrapperClassNames}>
         <select
           {...this.props.monthAttributes}
-          aria-label={label}
+          aria-label={this.props.isInvalid ? 'Error message' : label}
```

Comment:
This should probably be internationalized.

Reply:
The string "Error message" is not required so it has been removed.